fastlane/lanes/release.rb

```diff
@@ -363,7 +363,7 @@
     UI.important "Publishing release #{version_number} on GitHub"
 
     publish_github_release(
-      repository: GITHUB_REPO,
+      repository: GHHELPER_REPO,
```

Comment:
From reviewing https://github.com/wordpress-mobile/WordPress-Android/pull/21636, this change appears incorrect.

Reply:
This change was required by `25.7` ([see PR](https://github.com/wordpress-mobile/WordPress-Android/pull/21633/files)), but later `trunk` has been changed to use `GITHUB_REPO`, so we should avoid changing it back again.
```suggestion
      repository: GITHUB_REPO,
```